src/wallet/wallet2.cpp

```diff
@@ -9822,81 +9822,75 @@ static ons_prepared_args prepare_tx_extra_oxen_name_system_values(
                                 wallet.nettype(), *backup_owner, result.backup_owner, reason))
         return {};
 
-    {
-        nlohmann::json req_params{
-                {"entries",
-                 {{"name_hash", oxenc::to_base64(tools::view_guts(result.name_hash))},
-                  {"types", std::vector<uint16_t>{ons::db_mapping_type(type)}}}},
-        };
-        auto [success, response_] = wallet.ons_names_to_owners(req_params);
-        if (!response)
-            response = &response_;
-        else
-            *response = std::move(response_);
-        if (!success) {
+    nlohmann::json req_params{
+            {"name_hash", std::array{oxenc::to_base64(tools::view_guts(result.name_hash))}},
+            {"types", std::vector<uint16_t>{ons::db_mapping_type(type)}}};
```

Comment:
As it stands, `ons_info` takes 1 item, so this needs to be changed. If `ons_info` turns into a bulk API then this can remain unchanged.

Types is no longer accepted, this is going to return all the mappings for the hash so the body of this logic will have to change to lookup the type that's returned for the particular operation we're trying to perform on the ONS record.

Reply:
Yeah, `type` is indeed needed, and I've re-added it and updated this code to de-array the arguments.